packages/components/src/index.ts

```diff
@@ -1258,6 +1258,7 @@ export {
     mountWithServerContextOptions,
     // Ontology
     OntologyBrowserPanel,
+    OntologyBrowserPage,
```

Comment:
I don't think that the `OntologyBrowserPanel` component is not being used anymore outside of the @labkey/components package. I believe it can be removed from this index.ts in that case.